tests/e2e/vmgroups_test.go

```diff
@@ -113,7 +113,7 @@ func (vg *VmGroupTest) TearDownSuite(c *C) {
 
 	cmd := adminconst.ListVMgroups
 	out, err = ssh.InvokeCommand(vg.config.EsxHost, cmd)
-	log.Printf(out)
+	// log.Printf(out)
```

Comment:
Ok, I'd added this earlier but with all the tests out there this may be is not needed any more except for debug.